plugins/py3embed/py3embed.cpp

```diff
@@ -48,6 +48,11 @@
   #define Py_TPFLAGS_HAVE_ITER 0
 #endif
 
+#if PY_MINOR_VERSION < 7
+  #define USE_CUSTOM_NAMEDTUPLES
+#endif
+  #define USE_CUSTOM_NAMEDTUPLES
```

Comment:
is this meant to be here?

Reply:
No, just for me testing with 3.7. I will remove when I squash